test/fixtures/pretty/formatters.js

```diff
@@ -0,0 +1,13 @@
+global.process = { __proto__: process, pid: 123456 }
+Date.now = function () { return 1459875739796 }
+require('os').hostname = function () { return 'abcdefghijklmnopqr' }
+var pino = require(require.resolve('./../../../'))
+var log = pino({
+  prettyPrint: true,
+  formatters: {
+    log (obj) {
```

Comment:
I _think_ there is a msg property here with the message. You’d have to pass it through. Note that prettyPrint does not know anything about the new format, it just look for a given property.

Reply:
Thanks for your suggestion! The msg property wasn't available on that context, but I have pushed a simple change to make it available.  
Had to comment-out an assert  on formatters tests though, not sure about its significance.

Reply:
Scratch previous comment, I think I figured out the proper way for handling this.

Reply:
You don't have access to the message inside the `log` formatter:
https://github.com/pinojs/pino/blob/30d4b7d24425c7b5c133d241e1f684eb0552822d/lib/tools.js#L108-L113

If needed we can change this behavior.